packages/orchestration/README.md

```diff
@@ -17,16 +17,145 @@ $ npm install @sap-ai-sdk/orchestration
   - Create a `.env` file in the sample-code directory.
   - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`.
```

Comment:
[req] Seems like this does not belong in the official orchestration client docs?

Reply:
This is a part of all client documentation, until we create a common document applicable for all clients, I would let it be here.